scu-de/src/fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2/SwissbitCloudV2SCUConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2
+{
+    public class SwissbitCloudV2SCUConfiguration
+    {
+        public string TseSerialNumber { get; set; }
+        public string TseAccessToken { get; set; }
+        public bool EnableTarFileExport { get; set; } = true;
+        public virtual string CertificationId { get; set; } = "TBD";
+        public bool DisplayCertificationIdAddition { get; set; } = false;
+        public string CertificationIdAddition { get; set; }
+        public string ApiEndpoint { get; set; } = "https://dev.web-tse.de";
```

Comment:
This should probably be a different default URL in production

Reply:
Tom states there is no production url yet and the product is only available in sandbox with feature flag.